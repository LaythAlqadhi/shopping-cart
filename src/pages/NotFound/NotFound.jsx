import css from './NotFound.module.css';

const NotFound = () => {
  return (
    <div className={css.container}>
      <h1>404 - Page Not Found</h1>
      <p>We're sorry, but the page you are looking for cannot be found.</p>
    </div>
  );
}

export default NotFound;