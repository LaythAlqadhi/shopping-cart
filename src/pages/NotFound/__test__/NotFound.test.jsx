import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import userEvent from '@testing-library/user-event';
import NotFound from '../NotFound';

describe('NotFound component', () => {
  it('renders loading element correctly', () => {
    const { container } = render(<NotFound />)

    expect(container).toMatchSnapshot();
  })
})