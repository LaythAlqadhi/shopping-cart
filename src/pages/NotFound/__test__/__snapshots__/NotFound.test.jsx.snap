// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound component > renders loading element correctly 1`] = `
<div>
  <div
    class="_container_a04e95"
  >
    <h1>
      404 - Page Not Found
    </h1>
    <p>
      We're sorry, but the page you are looking for cannot be found.
    </p>
  </div>
</div>
`;
